solutions/remotemonitoring/single-vm/docker-compose.java.yml

```diff
@@ -69,23 +69,39 @@ services:
       - APPLICATION_SECRET
 
   devicesimulation:
-    image: azureiotpcs/device-simulation-dotnet:DS-1.0.2
+    image: azureiotpcs/device-simulation-dotnet:DS-2.0.1
     networks:
       - default_net
+    restart: always
     depends_on:
       - storageadapter
     environment:
+      - PCS_LOG_LEVEL
       - PCS_IOTHUB_CONNSTRING
       - PCS_STORAGEADAPTER_WEBSERVICE_URL=http://storageadapter:9022/v1
       - PCS_AUTH_ISSUER
       - PCS_AUTH_AUDIENCE
       - PCS_AUTH_REQUIRED
       - PCS_CORS_WHITELIST
-      - PCS_APPLICATION_SECRET
-      - APPLICATION_SECRET
-    # How one could mount custom device models
-    #volumes:
-    #  - ./my-device-models:/app/data:ro
+      - PCS_SUBSCRIPTION_ID
+      - PCS_WEBUI_AUTH_AAD_APPID=${PCS_AAD_APPID}
+      - PCS_WEBUI_AUTH_AAD_TENANT=${PCS_AAD_TENANT}
+      - PCS_AAD_CLIENT_SP_ID=${PCS_AAD_APPID}
+      - PCS_AAD_SECRET=${PCS_AAD_APPSECRET}
+      - PCS_RESOURCE_GROUP=${PCS_SOLUTION_NAME}
+      - PCS_IOHUB_NAME=${PCS_IOTHUB_NAME}
```

Comment:
nit: is it a typo `PCS_IOHUB_NAME`?

Reply:
This is intentional as the simulation service has a typo in the environment variable. See here:
https://github.com/Azure/device-simulation-dotnet/blob/0ced7244642ce2872ff66cc395fa8c60d0b90eef/WebService/appsettings.ini#L213